LKG builds are failing

```
src/tsc/tsc.ts(2,4): error TS2339: Property 'executeCommandLine' does not exist on type 'typeof ts'.
[06:05:47] 'buildTsc' errored after 1 min
[06:05:47] Error: Process exited with code: 2
    at ChildProcess.proc.on.exitCode (/home/vsts/work/1/s/scripts/build/utils.js:54:28)
    at ChildProcess.emit (events.js:198:13)
    at ChildProcess.EventEmitter.emit (domain.js:466:23)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
```

See also #34582.

--- a/src/tsc/tsconfig.release.json
+++ b/src/tsc/tsconfig.release.json
@@ -1,5 +1,5 @@
 {
-    "extends": "../tsconfig-base",
+    "extends": "../tsconfig.json",
     "compilerOptions": {
         "outFile": "../../built/local/tsc.release.js",
         "stripInternal": true,
@@ -10,9 +10,6 @@
         "composite": false,
         "incremental":  true
     },
-    "files": [
-        "tsc.ts"
-    ],
     "references": [
         { "path": "../compiler/tsconfig.release.json", "prepend": true }
     ]

